### ISSUE TITLE
Problem about reproduce the mot17 result

### ISSUE THREAD

**Issue by carry-xz** [bochinski/iou-tracker#10]
hi nice work!
I want to reproduce the mot17 track results. the result of mot17 FRCNN in README.txt is the average result  of all FRCNN?  because there are many FRCNN gt and det.

**Comment by bochinski** [comment]
yes, the FRCNN row shows the results if you only evaluate using the FRCNN detections on the mot17 sequences. Note that the test results include the evaluation results of all three detectors according to the mot17 evaluation protocol.
